File: Lab6/t3.cpp
#include <stdio.h>
int main(){
    int i, count = 0;
    char sentence[1000];
    scanf("%[^\n]", sentence);
    char* forSentence = sentence;
    for (i = 0; i < 1000; i++){
        if (*(forSentence+i) == '\n')
            break;
        if (*(forSentence+i) == 'a')
            count++;
    }
    printf("%d\n", count);
    return 0;
}

File: Lab6/t3.java
import java.util.Scanner;
public class t3{
    Scanner scan = new Scanner(System.in);
    String toCheck;
    System.out.println("Please input a sentence: ");
    int count = 0;
    toCheck = scan.nextLine();
    scan.close();
    char[] checkOut = toCheck.toCharArray();
    for (char a : checkOut){
        if (a == 'a')
            count++;
    }
    System.out.println(count);
}